ports/sonic.mania/Sonic Mania.sh

```diff
@@ -35,34 +35,28 @@ $ESUDO chmod 666 /dev/tty1
 $ESUDO chmod 777 $GAMEDIR/sonicmania
 
 # Modify PixWidth
-MED=320 # 4:3
+MED=320  # 4:3
 HIGH=424 # 16:9
 
 # Calculate the aspect ratio as a floating-point number
 ASPECT=$(awk "BEGIN {print $DISPLAY_WIDTH / $DISPLAY_HEIGHT}")
 
 # Set WIDTH based on aspect ratio comparisons
-if awk "BEGIN {exit ($ASPECT > 1.3)}"; then
-    WIDTH=$HIGH
-elif awk "BEGIN {exit ($ASPECT <= 1.3)}"; then
-    WIDTH=$MED
-else
-    WIDTH=$MED
-fi
+WIDTH=$(awk "BEGIN {print ($ASPECT > 1.3 ? $HIGH : $MED)}")
 
-if grep -q "^PixWidth=[0-9]\+" "$GAMEDIR/settings.ini"; then
-    sed -i "s/^PixWidth=[0-9]\+/PixWidth=$WIDTH/" "$GAMEDIR/settings.ini"
-    sed -i "s/^PixWidth=[0-9]\+/fsWidth=$DISPLAY_WIDTH/" "$GAMEDIR/settings.ini"
-    sed -i "s/^PixWidth=[0-9]\+/fsHeight=$DISPLAY_HEIGHT/" "$GAMEDIR/settings.ini"
+if grep -q "^pixWidth=[0-9]\+" "$GAMEDIR/Settings.ini"; then
```

Comment:
I'm all but certain this comes down to a file system case sensitivity thing. During my refactor tests this worked, so this PR change should be equivalently tested across FAT32, exFAT, and ext4.

Reply:
Yeah I also was pretty confident the Settings.ini not working for me was down to file system case sensitivity on my side, but I guess it still makes sense to use the actual file casing.

As for the grep - this seems like a legitimate problem? File system sensitivity wont affect this right?

Reply:
Yes I concur with grep addition.